src/Migration.php

```diff
@@ -654,4 +739,57 @@ protected function _set_args($what, $alias, $value)
             $this->args[$what][$alias] = $value;
         }
     }
+
+    /**
+     * Create Model Class php file content based on storage table.
+     *
+     * @param string $table
+     * @param string $model
+     * @param string $id_field
+     * @param string $namespace
+     *
+     * @throws Exception
+     *
+     * @return string
+     */
+    public function getModelPHPCode(string $table, string $model, string $id_field = 'id', string $namespace = '\Your\Project\Models') :string
```

Comment:
Why this method is here?
It's of course interesting to have, but I don't see it's relation to atk4\schema.

Reply:
i removed the method and the related test